lib/shared_methods.rb

```diff
@@ -0,0 +1,20 @@
+module Record
```

Comment:
Since the module is named `Record`, I'd suggest renaming the file name to match the module name (similar to the convention with classes)

It could even be renamed to be slightly more specific, since it contains methods to convert records to plain model objects, it could be called something like `RepositoryMethods` or similar